nixos/modules/services/misc/nix-gc.nix

```
@@ -1,104 +1,99 @@
 { config, lib, ... }:
-
 let
   cfg = config.nix.gc;
+  inherit (lib)
+    mkEnableOption
+    mkRenamedOptionModule
+    mkOption
+    types
+    ;
 in
-
 {
+  imports = [
+    (mkRenamedOptionModule [ "nix" "gc" "automatic" ] [ "nix" "gc" "enable" ])
+  ];
+
+  options.nix.gc = {
+    enable = mkEnableOption "Automatically run the garbage collector at a specific time.";
```

This will be prefixed fix `Whether to enable`

```suggestion
    enable = mkEnableOption "" // {
      description = "Whether to automatically run the garbage collector at a specific time.";
    };
```